go/vt/vttablet/tabletserver/repltracker/writer_test.go

```diff
@@ -33,12 +33,17 @@ import (
 	"vitess.io/vitess/go/vt/vttablet/tabletserver/tabletenv"
 )
 
+const (
+	testKeyspaceShard = "test:0"
```

Comment:
Two nits:
- This constant seems to be used exactly once. What is the value of defining it outside of the function where it is used?
- The standard delimiter is `/` so that this would be `test/0`. It would be even better to use `test/-` which we have been moving to.

Reply:
The constant used to be used elsewhere -- but that has changed. Good catch. Now inside to the function and is `test/0`.